packages/css-library/src/stylesheets/utilities.scss

```diff
@@ -24,5 +24,134 @@ $utilities-package: map-collect(
   $u-width
 );
 
-// Generate all utility classes
+// Generate all utility classes from USWDS function
 @include render-utilities-in($utilities-package);
+
+// Generate medium screen breakpoint utility classes.
+$medium-screen-breakpoint: 768px;
+$breakpoint-class-name: 'medium-screen';
+
+@mixin generate-utilities-from-settings($collection) {
+  @each $property, $property-map in $collection { 
+    $u-base: map-get($property-map, base);
+    $u-values: map-get($property-map, values);
+    $u-property: map-get($property-map, property);
+    $u-modifiers: map-get($property-map, modifiers);
+
+    @if $u-modifiers {
+      @each $modifier, $modifier-value in $u-modifiers {
+        @each $name, $value in $u-values {
+          .#{$breakpoint-class-name}\:#{$u-base}#{$modifier-value}-#{$name} {
+            @if $modifier == 'noModifier' {
+              #{$u-property}: #{$value};
+            } @else {
+              #{$u-property}#{$modifier-value}: #{$value};
+            }
+          }
+        }
+      }
+    }
+
+    @each $name, $value in $u-values {
+      .#{$breakpoint-class-name}\:#{$u-base}-#{$name} {
+        #{$u-property}: #{$value};
+      }
+    }
+  }
+}
+
+@media all and (min-width: $medium-screen-breakpoint) {
+  // $u-border
+  @include generate-utilities-from-settings($u-border);
+
+  // $u-display
+  @include generate-utilities-from-settings($u-display);
+
+  // $u-flex-grid
+  @each $property, $property-map in $u-flex-grid { 
```

Comment:
The `u-flex-grid` and `u-margin` settings are more complicated packages that the others. So instead of continuing to build out the `generate-utilities-from-settings` logic, I decided to just give them their own sections.

All of this will hopefully be removed anyway once USWDS core is updated to allow for custom breakpoints.